src/modules/configurator/components/saved-configurations/Configurations.tsx

```diff
@@ -0,0 +1,23 @@
+import ConfigureLinkButton from "../ConfigureLinkButton";
+import ConfigurationsEmpty from "./ConfigurationsEmpty";
+import { useRecoilValue } from "recoil";
+import { userConfigurationsState } from "../../state/configurationsState";
+import ConfigurationsSaved from "./ConfigurationsSaved";
+
+export default function Configurations() {
+  const userConfigurations = useRecoilValue(userConfigurationsState);
+
+  return (
+    <section className="text-center md:text-left px-10 sm:px-20 md:px-20 lg:px-36 py-16 flex flex-col">
+      <header className="flex flex-col sm:flex-row justify-between">
+        <h2 className="mb-6 text-lg sm:text-sm">View saved configurations</h2>
+        <ConfigureLinkButton path="/configure-a-car" title="Configure a car" />
+      </header>
+      {userConfigurations.length === 0 ? (
```

Comment:
You can also use `!!userConfigurations.length ?` or `Boolean(userConfigurations.length) ?`